### DISCUSSION TITLE
fitler when no files on a query

### DISCUSSION THREAD

**Discussion by pagoru** [pocketbase/pocketbase#4768] [Q&A] [answered]
I'm trying to filter when no files are present (N/A). I'm trying with COUNT(files) > 0 but it doesn't work as expected. There's another way to do it? Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure I follow but you could check https://github.com/pocketbase/pocketbase/discussions/4757.

If that doesn't help, please elaborate a little more on your use case and your collections structure (_Admin UI > Settings > Export collections_).

**Reply by pagoru** [reply]
Exactly that! 
`picture != "[]"`

Thanks as always!
